case-projector/src/main/java/eu/itesla_project/case_projector/CaseProjectorUtils.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.case_projector;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.powsybl.ampl.converter.*;
+import com.powsybl.commons.datasource.FileDataSource;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.computation.*;
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.Generator;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.loadflow.LoadFlow;
+import com.powsybl.loadflow.LoadFlowParameters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>
+ */
+public final class CaseProjectorUtils {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CaseProjectorUtils.class);
+
+    private static final Set<String> AMPL_MODEL_FILE_NAMES = ImmutableSet.<String>builder()
+            .add("projector.run")
+            .add("projector.mod")
+            .add("projector.dat")
+            .add("projectorOutput.run")
+            .build();
+
+    private static final String WORKING_DIR_PREFIX = "itesla_projector_";
+
+    private static final String AMPL_GENERATORS_DOMAINS_FILE_NAME = "ampl_generators_domains.txt";
+
+    private static final LoadFlowParameters LOAD_FLOW_PARAMETERS = LoadFlowParameters.load();
+
+    private static final LoadFlowParameters LOAD_FLOW_PARAMETERS2 = LoadFlowParameters.load().setNoGeneratorReactiveLimits(true);
+
```

Comment:
I know it was not part of that evolution, but I would prefer the parameters (1 and 2 above) to be lazily initialized when the projector is run.
--> can you remove those 2 fields and make them local variables in the "project" method?
They may be reloaded more often but it's not a problem.